gapis/api/vulkan/vulkan.go

```diff
@@ -29,9 +29,11 @@ import (
 )
 
 type CustomState struct {
-	SubcommandIndex   sync.SubcommandIndex
+	SubcommandIndex   api.SubcommandIndex
 	CurrentSubmission *api.Cmd
 	HandleSubcommand  func(interface{}) `nobox:"true"`
+	AddCommand        func(interface{}) `nobox:"true"`
```

Comment:
Given that the `CustomState` field is `nobox:"true"`, I'm surprised you need it here.